types/react-joyride/index.d.ts

```diff
@@ -9,15 +9,11 @@ import React = require("react");
 export default class Joyride extends React.Component<Props, State> {
     constructor(props: Props);
 
-    addTooltip(data: Step): void;
-    start(autorun?: boolean): void;
```

Comment:
Nit: instead of removing `start` and `stop`, maybe these should be marked with `/** @deprecated */` or `private`.